### PR TITLE
Update blog page cover, add layer

### PR DESCRIPTION
**Pull request by hectoruch**
Update blog page cover, add layer

### CODE REVIEW

File: app/src/components/shared/CoverPrimary.jsx [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
 import LogoLDF from '../../../assets/logos/ldf_logo.png';
+import Header from '../../containers/header';
+import MenuMobile from '../shared/menu_mobile';
 
 class CoverPrimary extends Component {
 
   render() {
     return (<section className={CoverPage['c-cover-page']}>
-      <div>
+      <div className={CoverPage['layer-cover']}>
```
Comment (tiagojsag) [inline app/src/components/shared/CoverPrimary.jsx]: this breaks other pages that are reusing this component.

File: app/src/components/shared/CoverPrimary.jsx [hunk 2]
```diff
@@ -15,6 +19,7 @@ class CoverPrimary extends Component {
         </p>
         <div className={CoverPage['footer-header']}>
           <div>
+            <p className={CoverPage['text-ldf-footer']}>Brought to you by:</p>
```
Comment (tiagojsag) [inline app/src/components/shared/CoverPrimary.jsx]: why did you add this back? remove it

File: app/src/components/shared/CoverSecondary.jsx [hunk 3]
```diff
@@ -1,19 +1,27 @@
...
         </h1>
         <p className={CoverPage['cover-sub-subtitle']}>
-          {this.props.subtitle}
+            {this.props.subtitle}
         </p>
-        <div className={CoverPage.footer_header_blog}>
-          <div>Brought to you by:</div>
+        <div className={CoverPage['footer-header']}>
+          <div>
+            <p className={CoverPage['text-ldf-footer']}>Brought to you by:</p>
```
Comment (tiagojsag) [inline app/src/components/shared/CoverSecondary.jsx]: why did you add this back? remove it

**Comment by hectoruch** [comment]
Done! 
That code was delete again. @tiagojsag

**Comment by hectoruch** [comment]
Changes:

- New height on cover of 2nd level sections.
- New logo on menu of second level sections.
- Refactor scss file and add comments.

@clementprdhomme

**Comment by tiagojsag** [comment]
conflicts,

**Comment by hectoruch** [comment]
- Changes responsive styles on menu.
- Change the function to change background menu on different pages.

@tiagojsag @clementprdhomme
